about.html

```diff
@@ -81,57 +87,63 @@ <h2>South Korea</h2>
         <!--div 3 light-->
         <div class="bg-light text-dark p-2">
             <h2 class="text-center">See the past Developing Contest Global winners</h2>
-            <hr class="my-4 text-primary bg-primary border-5">
+            <hr>
             <p class="text-center"> Take a look at the last two developing contest global winner, celebrated in Rio do Janeiro and in Vienna</p>
             
-            <div class="card text-center p-2 hero-image text-light mb-2 tw-400">
-                <div class="card-body">
-                  <h3 class="card-title">2019</h3>
-                  <h6 class="card-subtitle mb-2">Winner: Japan</h6>
-                  <p class="card-text">Location: Brazil</p>
+            <div class="presenters">
+                <div class="card text-center hero-image text-light mb-2 tw-400">
+                    <div class="card-body hero-div">
+                      <h3 class="card-title mt-2">2019</h3>
+                      <h6 class="card-subtitle mb-2">Winner: Japan</h6>
+                      <p class="card-text mb-2">Location: Brazil</p>
+                    </div>
                 </div>
-            </div>
-
-            <div class="card text-center p-2 hero-image text-light mb-2 tw-400">
-                <div class="card-body">
-                  <h3 class="card-title">2018</h3>
-                  <h6 class="card-subtitle mb-2">Winner: Japan</h6>
-                  <p class="card-text">Location: Brazil</p>
+    
+                <div class="card text-center hero-image text-light mb-2 tw-400">
+                    <div class="card-body hero-div">
+                      <h3 class="card-title mt-2">2019</h3>
+                      <h6 class="card-subtitle mb-2">Winner: Japan</h6>
+                      <p class="card-text mb-2">Location: Brazil</p>
+                    </div>
                 </div>
             </div>
+            
 
         </div>
 
         <!--div 4 dark-->
         <div class="bg-dark text-light text-center">
             <br>
             <h2>Partner</h2>
-            <hr class="my-4 text-primary bg-primary border-5">
+            <hr>
             <div class="d-flex flex-wrap justify-content-center">
                 <h2 class="m-2">mozilla</h2>
                 <h2 class="m-2">Microsoft</h2>
                 <h2 class="m-2">Google</h2>
                 <h2 class="m-2">Alex Dev</h2>
                 <h2 class="m-2">Udemy</h2>
             </div>
+            <br>
         </div>
 
         <br>
         <!--footer light-->
         <footer class="bg-light text-dark">
-            <div class="d-flex flex-row p-1">
-                <div class="me-2">
+            <div class="footer-info">
+                <div class="me-2 footer-logo">
                     <img class="me-1 logo" src="./assets/Logo.svg" alt="">
```

Comment:
- [ ] Please add an alt value for every image on your project, this helps for SEO purposes and has a better qualification for google. 🪐